BenchmarkSample/benchmark/build.gradle

```diff
@@ -3,19 +3,20 @@ apply plugin: 'kotlin-android'
 apply plugin: 'androidx.benchmark'
 
 android {
-    compileSdkVersion 29
+    compileSdkVersion 31
 
     defaultConfig {
         minSdkVersion 14
-        targetSdkVersion 29
+        targetSdkVersion 31
 
         // Set this argument to capture profiling information, instead of measuring performance.
         // Can be one of:
         //   * None
```

Comment:
These have been renamed:

"Method" -> "MethodTracing"
"Sampled" -> "StackSampling"

Reply:
changed to what you mention